bin/pyenv-installer

```diff
@@ -22,7 +22,7 @@ colorize() {
 }
 
 checkout() {
-  [ -d "$2" ] || git clone "$1" "$2"
+  [ -d "$2" ] || git clone --depth 1 "$1" "$2"
```

Comment:
That's right, you can see the `git help fetch` manpage for instructions on
how to "un-shallow" a repo :)

On Thu, 14 Jul 2016 at 15:24 Yamashita, Yuu <notifications@github.com>
wrote:

> In bin/pyenv-installer
> <https://github.com/yyuu/pyenv-installer/pull/42#discussion_r70750096>:
>
> > @@ -22,7 +22,7 @@ colorize() {
> >  }
> >
> >  checkout() {
> > -  [ -d "$2" ] || git clone "$1" "$2"
> > +  [ -d "$2" ] || git clone --depth 1 "$1" "$2"
>
> Ah it seems I was misunderstanding. It seems we can git pull to fetch
> latest changes even with shallow clone repository. If it works fine, we can
> use --depth 1 by default.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/yyuu/pyenv-installer/pull/42/files/97339195061c2e2665519696c765e5dd29b4b98c#r70750096>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAXKdXu11Kw32gsgOdY71gdc5ILwm4m_ks5qVcgdgaJpZM4JMA6h>
> .
>